pkg/chat/model.go

```diff
@@ -25,6 +25,11 @@ Doctor's work can be reduced by automatically showing right questions.
 type Catagory struct {
 	ID        uint `gorm:"primaryKey"`
 	Name      string
-	Keywords  *string
-	Questions *string //Each catagory correspond with a set of fixed questions
+	Keywords  []Keyword `gorm:"many2many:catagory_keywords;"`
```

Comment:
You'll need the reversed direction. You get catagory by keywords, not get keywords by catagory